### ISSUE TITLE
[MAIN] [Pipeline] Model analysis

### ISSUE THREAD

**Issue by Mars-Martiny** [Mars-Martiny/CITS5206_group1#26]
Add standard functions used to compare model training and performance:
- plot relevant metrics from training loop,
- evaluate on the test set,  
- plot confusion matrix
- etc.

**Comment by AndreHrs** [comment]
This seems like already closed by PR #52 and #40

**Comment by Mars-Martiny** [comment]
Major integration fixes needed, instead of reopening will just keep new bug fix issue for this.
